src/main/javassist/bytecode/LocalVariableAttribute.java

```diff
@@ -351,4 +355,45 @@ public AttributeInfo copy(ConstPool newCp, Map<String,String> classnames) {
     LocalVariableAttribute makeThisAttr(ConstPool cp, byte[] dest) {
         return new LocalVariableAttribute(cp, tag, dest);
     }
+
+    /**
+     * Undocumented method. Do not use; internal-use only.
+     * 
+     * Returns a list of entries in the correct order according to the local
+     * variable index. This method is for internal use only as the order may be
+     * wrong with local variables with different type descriptor.
+     * 
+     * @see #index(int)
+     * @since 3.30
+     */
+    public List<Integer> entryListOrderedByIndex() {
+	int n = tableLength();
+	List<Integer> entries = new ArrayList<Integer>(Arrays.asList(new Integer[n]));
+	for (int i = 0; i < n; i++) {
+	    int index = index(i);
+	    while (index < entries.size() && entries.get(index) != null) {
+		index++;
+	    }
+
+	    if (index < n) {
+		entries.set(index, i);
+	    } else {
+		entries.add(i);
+	    }
+	}
+
+	if (entries.size() > n) {
```

Comment:
The index method can return the same integer for different local variables (you can run the test I wrote in the debugger to be sure). To prevent the list from having "null" elements, I put the conditional `if (entries.size() > n)`, but in practice the conditional is never executed in all branches of the internal libraries that I have access to or in the test that I I wrote. JVM is not my area of expertise, for this reason in the index method I am literally programming blindly and I am in doubt whether or not I should remove this conditional. With the exception of this conditional, the method works as expected.